hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -497,6 +496,27 @@
       milestone: V2
       tests: ["chip_jtag_csr_rw"]
     }
+    {
+      name: chip_sw_jtag_mem_access
+      desc: '''
+            Verify accessibility of all the memories in the chip over JTAG.
+
+            This test will target the following memories in the chip:
+              sram_main, sram_ret, ibex i|dcache, otbn i|dcache, ROM
```

Comment:
nit: otbn it's called imem / dmem

for the icache inside ibex, i don't actually think we have a direct path. Ie it would end up being an access through the processor.